### PR TITLE
Add configuration details to readme

### PR DESCRIPTION
**Pull request by richardmiller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,4 +178,59 @@ The `--namespace|-n` option can be used with the `watch`, `ssh`, `resync` and `f
...
+project specific naming convention for this e.g. `remote-<your name>`
+
+### REMOTE_NAME
+
+The name of the git remote for the GitHub project which has the Continuous Pipe integration.
+In most cases you will have cloned the project repo from this so this will be `origin`.   
+ 
+### DEFAULT_CONTAINER     
+ 
+This is an optional setting, if provided this will be used by the `ssh`, `watch` and `resync` commands.
```
Comment (tonypiper) [inline README.md]: worth adding a comment about what the default is?

File: README.md [hunk 2]
```diff
@@ -177,4 +178,59 @@ The `--namespace|-n` option can be used with the `watch`, `ssh`, `resync` and `f
...
+It is the docker-compose service name for the container that you need to provide, it may be called 
+something like `web` or `app`.
+
+### KEEN_WRITE_KEY, KEEN_PROJECT_ID and KEEN_EVENT_COLLECTION
+ 
+These are only needed if you want to log usage stats using https://keen.io/.
```
Comment (tonypiper) [inline README.md]: link the url?

File: README.md [hunk 3]
```diff
@@ -177,4 +178,59 @@ The `--namespace|-n` option can be used with the `watch`, `ssh`, `resync` and `f
...
+
+### KEEN_WRITE_KEY, KEEN_PROJECT_ID and KEEN_EVENT_COLLECTION
+ 
+These are only needed if you want to log usage stats using https://keen.io/. 
+
+### ANYBAR_PORT
+
+This is only needed if you want to get [AnyBar](https://github.com/tonsky/AnyBar) notifications.
+A value needs to be provided, even if you want to use the default port of 1738, as the 
+notification are not sent when no port number is provided.
```
Comment (tonypiper) [inline README.md]: notifications

File: README.md (hunk 2 above)
Comment (richardmiller) [inline README.md]: This should be automatically turned into link when viewing the readme on github.

File: README.md [hunk 4]
```diff
@@ -177,4 +178,60 @@ The `--namespace|-n` option can be used with the `watch`, `ssh`, `resync` and `f
...
+ 
+### PROJECT_KEY
+ 
+This is the project name used in Continuous Pipe. It will be prefixed to all the environment
+names created by Continuous Pipe. You can find this on the environments page for the tide on the 
+[Continuous Pipe admin site](https://ui.continuouspipe.io/). For example:
+
+![Project Key](/docs/images/project-key.png?raw=true)
+
+Here, this is the environment for the develop branch, so the project key is `develop`.
```
Comment (sroze) [inline README.md]: Isn't it... `my-project` ? 😉

File: README.md [hunk 5]
```diff
@@ -177,4 +178,60 @@ The `--namespace|-n` option can be used with the `watch`, `ssh`, `resync` and `f
...
+
+### ANYBAR_PORT
+
+This is only needed if you want to get [AnyBar](https://github.com/tonsky/AnyBar) notifications.
+A value needs to be provided, even if you want to use the default port of 1738, as the 
+notifications are not sent when no port number is provided.
+
+### Kubernetes settings
+ 
+These are asked for by the `set up` command but are not stored in the project config file. The
```
Comment (tonypiper) [inline README.md]: `setup`

**Comment by sroze** [comment]
🎉

**Pull request merged**